ports/nrf/common-hal/alarm/__init__.c

```diff
@@ -48,6 +48,9 @@
 #include "nrf_power.h"
 #include "nrfx.h"
 #include "nrfx_gpiote.h"
+#ifdef NRF_DEBUG_PRINT
+static void print_wakeup_cause(nrf_sleep_source_t cause);
+#endif
```

Comment:
Sorry, I should have been more specific. The function below can be static and then this bit isn't needed.